output.cpp

```diff
@@ -60,7 +61,7 @@ void Output::OutputReady(void *mem, size_t size, int64_t timestamp_us, bool keyf
 
 	// Save timestamps to a file, if that was requested.
 	if (fp_timestamps_)
-		fprintf(fp_timestamps_, "%lld.%03lld\n", last_timestamp_ / 1000, last_timestamp_ % 1000);
+		fprintf(fp_timestamps_, PRId64 ".%03" PRId64 "\n", last_timestamp_ / 1000, last_timestamp_ % 1000);
```

Comment:
This is missing a `"%"` before the first `PRId64`, no?

Reply:
Yes, good catch!

@davidplowman can you force push this fix. Sorry about missing that.

Reply:
Will do. I might add something for this to the "smoke tests", that is, test.py. In general I'm keen to keep beefing those up as we go...